nodejs/src/utils/QueryResultsIterator.ts

```diff
@@ -0,0 +1,23 @@
+import { QueryResults } from "../apis/interfaces/common";
+
+export class QueryResultsIterator<T> {
+  private results!: QueryResults<T>;
```

Comment:
let's not use not null operator and add a condition instead at the time of use if the variable can be null